### PR TITLE
Implement GET SubscriptionCollection

### LINKED ISSUE #212: Get Subscription collection
As oneview-redfish-toolkit user, I send a GET request to "/redfish/v1/EventService/EventSubscriptions" and I receive a JSON containing a collection of subscriptions created.

### PR DESCRIPTION
**Pull request by llvieira**
Closes #212.

### CODE REVIEW

File: oneview_redfish_toolkit/api/subscription_collection.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        super().__init__(self.SCHEMA_NAME)
+
+        self.redfish["@odata.type"] = \
+            "#EventDestinationCollection.EventDestinationCollection"
+        self.redfish["Name"] = "Event Subscriptions Collection"
+        self.redfish["Members@odata.count"] = 1
+        self.redfish["Members"] = list()
+        self.redfish["Members"].append(collections.OrderedDict())
+        # This information is currently hard-coded
+        self.redfish["Members"][0]["@odata.id"] = \
```
Comment (ricardoas) [inline oneview_redfish_toolkit/api/subscription_collection.py]: Not hard coded, please!!!! :(
At this point we can define at lease empty dictionaries in utils to hold collections of subscriptions and iterate on them here... they'll be empty on demos until we implement add subscription endpoints but that's life...

File: oneview_redfish_toolkit/blueprints/subscription_collection.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+from flask import Response
+from flask_api import status
+
+from oneview_redfish_toolkit.api.subscription_collection \
+    import SubscriptionCollection
+
+subscription_collection = Blueprint("subscription_collection", __name__)
+
+
+@subscription_collection.route(
```
Comment (ricardoas) [inline oneview_redfish_toolkit/blueprints/subscription_collection.py]: I haven't found any info about this but what is supposed to happen when we try to access/modify this collection in a scenario that `EventService['ServiceEnabled'] == False`????? Should we fail all requests?

File: oneview_redfish_toolkit/blueprints/subscription_collection.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    """Get the Redfish Subscription Collection.
+
+        Get method to return SubscriptionCollection JSON when
+        /redfish/v1/SessionService/EventSubscriptions is requested.
+        Returns:
+                JSON: JSON with EventSubscriptions.
+    """
+    try:
+        # Build Subscription Collection object and validates it
+        sc = SubscriptionCollection()
```
Comment (ricardoas) [inline oneview_redfish_toolkit/blueprints/subscription_collection.py]: that dictionary would be passed here...

File: oneview_redfish_toolkit/api/subscription_collection.py (hunk 1 above)
Comment (llvieira) [reply oneview_redfish_toolkit/api/subscription_collection.py]: Ok, I'm going to change that.

File: oneview_redfish_toolkit/blueprints/subscription_collection.py (hunk 3 above)
Comment (llvieira) [reply oneview_redfish_toolkit/blueprints/subscription_collection.py]: But, if the dicts are in `util.py` I can retrieve them directly at `SubscriptionCollection` class. Is it better to pass them there or catch it directly in the class?

File: oneview_redfish_toolkit/blueprints/subscription_collection.py (hunk 2 above)
Comment (llvieira) [reply oneview_redfish_toolkit/blueprints/subscription_collection.py]: I didn't find any info too, but fail all requests when `ServiceEnabled == False` looks good for me.

Comment (llvieira) [reply oneview_redfish_toolkit/blueprints/subscription_collection.py]: @ricardoas In that case of fail the requests, should I do a GET operation by the `EventService` into `SubscriptionCollection` to check if `ServiceEnabled == False`???

File: oneview_redfish_toolkit/blueprints/subscription_collection.py (hunk 3 above)
Comment (llvieira) [reply oneview_redfish_toolkit/blueprints/subscription_collection.py]: Done as you said! :)

File: oneview_redfish_toolkit/blueprints/subscription_collection.py (hunk 2 above)
Comment (ricardoas) [reply oneview_redfish_toolkit/blueprints/subscription_collection.py]: I guess that if we have this logic isolated in a function in util we can bypass the GET operation and just call the method directly... wdyt?

File: oneview_redfish_toolkit/blueprints/subscription_collection.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+from oneview_redfish_toolkit.api.subscription_collection \
+    import SubscriptionCollection
+from oneview_redfish_toolkit import util
+
+
+subscription_collection = Blueprint("subscription_collection", __name__)
+
+
+@subscription_collection.route(
+    "/redfish/v1/SessionService/EventSubscriptions/", methods=["GET"])
```
Comment (ricardoas) [inline oneview_redfish_toolkit/blueprints/subscription_collection.py]: This should be: "/redfish/v1/EventService/EventSubscriptions/"

File: oneview_redfish_toolkit/api/subscription_collection.py [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+            all_subscriptions: Dictionary with all subscriptions.
+        """
+        super().__init__(self.SCHEMA_NAME)
+
+        self.redfish["@odata.type"] = \
+            "#EventDestinationCollection.EventDestinationCollection"
+        self.redfish["Name"] = "Event Subscriptions Collection"
+        self.redfish["Members@odata.count"] = len(all_subscriptions)
+        self.redfish["Members"] = list()
+        self.redfish["Members"].append(collections.OrderedDict())
```
Comment (ricardoas) [inline oneview_redfish_toolkit/api/subscription_collection.py]: Is this a ghost member of this collection?

Comment (llvieira) [reply oneview_redfish_toolkit/api/subscription_collection.py]: HAHA, just a trash.

File: oneview_redfish_toolkit/blueprints/subscription_collection.py (hunk 2 above)
Comment (llvieira) [reply oneview_redfish_toolkit/blueprints/subscription_collection.py]: Sounds good!

File: oneview_redfish_toolkit/api/subscription_collection.py (hunk 5 above)
Comment (llvieira) [reply oneview_redfish_toolkit/api/subscription_collection.py]: Fixed!

**Pull request merged**